README.md

```diff
@@ -12,10 +12,13 @@ Universum
 
 1. **Excellent documentation**: tutorial, migration guide from `Prelude`,
    Haddock with examples for (almost) every function,
-   all examples are tested with [`doctest`](http://hackage.haskell.org/package/doctest).
+   all examples are tested with [`doctest`](http://hackage.haskell.org/package/doctest),
+   documenation regarding internal module structure.
 2. `universum`-specific [HLint](http://hackage.haskell.org/package/hlint) rules:
    [`.hlint.yaml`](https://github.com/serokell/universum/blob/master/.hlint.yaml)
-3. Focus on safety, convenience and efficiency.
+3. Very few LiquidHaskell properties. At least LiquidHaskell is on Travis CI and
```

Comment:
"Only a few LiquidHaskell properties right now, but LiquidHaskell is on Travis CI and other properties are just waiting to be added!" ?